packages/util/src/bytes.ts

```diff
@@ -386,3 +386,7 @@ export const bigIntToHex = (num: bigint) => {
 export function bigIntToUnpaddedBuffer(value: bigint): Buffer {
   return unpadBuffer(bigIntToBuffer(value))
 }
+
```

Comment:
Won't make this a blocker but generally please always document methods which become part of the public API.

Have added to the tiny-fixes-issue #2105 so that we won't forget.